docs/extras.rst

```diff
@@ -21,7 +21,7 @@ hypothesis-datetime lives in the hypothesis.extra.datetime package:
 
 .. code-block:: pycon
 
-  >>> from datetime import datetime
+  >>> from datetime import datetimes
```

Comment:
Should import source be `hypothesis.extra.datetime`? Because given example still will fail if I just re-type it in REPL.